src/Renci.SshNet/Security/Cryptography/Ciphers/AesGcmCipher.cs

```diff
@@ -62,10 +62,18 @@ public AesGcmCipher(byte[] key, byte[] iv, int aadLength)
             // SSH AES-GCM requires a 12-octet Initial IV
             _iv = iv.Take(12);
             _aadLength = aadLength;
-#if NET
+#if !NETSTANDARD
             if (System.Security.Cryptography.AesGcm.IsSupported)
             {
-                _impl = new BclImpl(key, _iv);
+                try
+                {
+                    _impl = new BclImpl(key, _iv);
+                }
+                catch (DllNotFoundException)
```

Comment:
have you hit this? I don't think we need to be that worried about it anymore

Reply:
The CI test was ran at Ubuntu and hit this issue before.
I can unwrap "try catch" if you want.